app/assets/javascripts/analytics/static-tracker.js

```diff
@@ -78,6 +79,26 @@
       tracker.setDimension(customVar[1], customVar[3], customVar[2], customVar[4]);
     }
 
+    function setDimensionsFromMetaTags() {
```

Comment:
I saw no reason to add `setDimensionsFromMetaTags` to the public api as its only use is during initialisation.